htdocs/mrp/ajax/interface.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ *	\file       htdocs/mrp/ajax/interface.php
+ */
+
+if (!defined('NOREQUIRESOC')) {
+	define('NOREQUIRESOC', '1');
+}
+if (!defined('NOCSRFCHECK')) {
+	define('NOCSRFCHECK', '1');
+}
+if (!defined('NOTOKENRENEWAL')) {
+	define('NOTOKENRENEWAL', '1');
+}
+if (!defined('NOREQUIREMENU')) {
+	define('NOREQUIREMENU', '1');
+}
+if (!defined('NOREQUIREHTML')) {
+	define('NOREQUIREHTML', '1');
+}
+if (!defined('NOREQUIREAJAX')) {
+	define('NOREQUIREAJAX', '1');
+}
+
+global $db, $langs;
+
+require '../../main.inc.php'; // Load $user and permissions
+
```

Comment:
Can you add here this:

top_httphead("text/json");
I let you see the content of this metho to understand why...